aiida_common_workflows/workflows/relax/bigdft/workchain.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""Implementation of `aiida_common_workflows.common.relax.workchain.CommonRelaxWorkChain` for BigDFT"""
+from aiida.plugins import WorkflowFactory
+
+from ..workchain import CommonRelaxWorkChain
+
+__all__ = ('BigDFTCommonRelaxWorkChain',)
+
+RelaxWorkChain = WorkflowFactory('bigdft.relax')
+
+
+class BigDFTCommonRelaxWorkChain(CommonRelaxWorkChain):
+    """Implementation of `aiida_common_workflows.common.relax.workchain.CommonRelaxWorkChain` for BigDFT."""
+
+    _process_class = RelaxWorkChain
+
+    @classmethod
+    def define(cls, spec):
+        # yapf: disable
+        super().define(spec)
+        spec.expose_outputs(RelaxWorkChain)
```

Comment:
This is fine for now, but will break as soon as the spec of `bigdft.relax` changes. I will add a test in the future to automatically detect this. Just something to be aware of for the future. We can leave it as is for now